### PR TITLE
CLOUD-310 PXC documentation update

### PR DESCRIPTION
**Pull request by cap1984**
Some step shall be updated due to typos or incorrect logic.

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona/percona-xtradb-cluster-operator?pullRequest=204) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/percona/percona-xtradb-cluster-operator?pullRequest=204) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/percona/percona-xtradb-cluster-operator?pullRequest=204) it.</sub>

File: source/kubernetes.rst [hunk 1]
```diff
@@ -74,7 +74,7 @@ Install Percona XtraDB Cluster on Kubernetes
    More details about secrets can be found in a `separate
    section <../configure/users>`__.
 
-5. Now you need to `prepare certificates for TLS security <TLS.html>`_ and apply them with the following command:
+5. Install `cert-manager <https://docs.cert-manager.io/en/release-0.8/getting-started/install/kubernetes.html>`_ if it is not up and running yet then generate and apply certificates as secrets according to `TLS document <TLS.html>`:
 
    .. code:: bash
```
Comment (delgod) [inline source/kubernetes.rst]: could we please put line 83 before line 81

Comment (cap1984) [reply source/kubernetes.rst]: done

File: source/openshift.rst [hunk 2]
```diff
@@ -79,7 +79,7 @@ Install Percona XtraDB Cluster on OpenShift
    More details about secrets can be found in a `separate
    section <../configure/users>`__.
 
-5. Now you need to `prepare certificates for TLS security <TLS.html>`_ and apply them with the following command:
+5. Install `cert-manager <https://docs.cert-manager.io/en/release-0.8/getting-started/install/openshift.html>`_ if it is not up and running yet then generate and apply certificates as secrets according to `TLS document <TLS.html>`:
```
Comment (delgod) [inline source/openshift.rst]: Could you please put line 88 before line 86

Comment (cap1984) [reply source/openshift.rst]: done

**Pull request merged**